components/Footer/Footer.test.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Footer from "./Footer";
+
+test("Renders the footer", () => {
+  const { getByText } = render(<Footer />);
```

Comment:
You should import `screen` then use `screen.getByText` instead. This is now the recommended way to use react-testing rather than de-structuring utils from the render func. Most of the utils you're used to using are available on `screen`.

```js
import { render, screen } from "@testing-library/react";

render(<Footer />);

screen.getByText(TEXT);
```

Reply:
refactored.